modules/new_user/new_user.py

```diff
@@ -0,0 +1,47 @@
+import os
+from discord.utils import get
+import csv
+
+def is_unassigned(member):
+	#if the member has the unassigned role
+	return (get(member.roles, id=int(get_id('UNASSIGNED_ROLE_ID'))) != None)
+
+
+async def add_role(ctx, machon, year):
+	'''adds the right role to the user that used the command'''
+	member = ctx.author
+
+	#formatting role for env file
+	role_id = machon.upper() + '_YEAR_' + year + "_ROLE_ID" #format example: LEV_YEAR_1_ROLE_ID
+	new_role = get(member.guild.roles, id = int(get_id(role_id)))
+
+	#check if the role was legal
+	if new_role == None:
+		await ctx.send("The Machon or year was wrong. Please try again or contact an admin for help.")
+		return
+    
+	await member.add_roles(new_role)
+	print(f"Gave {new_role} to {member}")
+    
+
+async def remove_unassigned(member):
+	'''removes the unassigned role from member'''
+	role = get(member.guild.roles, id = int(get_id("UNASSIGNED_ROLE_ID")))
+	await member.remove_roles(role)
+	print(f"Removed Unassigned from {member}")
+    
+    
+async def change_nick(member, first_name, last_name):
+		'''changes the nick name of the member to their full name'''
+		name = first_name.capitalize() + ' ' + last_name.capitalize()
+		await member.edit(nick = name)
+		print(f"Set {member} nick to {name}")
+
+def get_id(name):
```

Comment:
Might as well return it as an integer instead of converting to int in a bunch of places